<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/info_icon.png</normaloff>:/images/info_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextBrowser {
	font-size: 16px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="FileInfo">
      <attribute name="title">
       <string>FileInfo</string>
      </attribute>
      <widget class="QTextBrowser" name="textBrowser_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Help">
      <attribute name="title">
       <string>Help</string>
      </attribute>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="AuthorInfo">
      <attribute name="title">
       <string>AuthorInfo</string>
      </attribute>
      <widget class="QTextBrowser" name="textBrowser_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
